<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Управление терминальными подключениями</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/ikonka/image/opensuse-logo.png</normaloff>:/ikonka/image/opensuse-logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QTreeWidget" name="twg">
      <column>
       <property name="text">
        <string>Пользователь</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Xorg</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sesman</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>№ DISPLAY</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Время подключения</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Активность</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Простой</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Сообщение</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Текст сообщения</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Выход</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/ikonka/image/down.png</normaloff>:/ikonka/image/down.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Фильтр процессов</string>
      </property>
      <widget class="QCheckBox" name="checkBox_proc">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>0</y>
         <width>88</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>вкл</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_proc">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>0</y>
         <width>113</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>1c</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QTreeWidget" name="twg2">
      <column>
       <property name="text">
        <string>PID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Статус</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Время</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>БД 1С</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>946</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Управление</string>
    </property>
    <addaction name="action_refresh"/>
    <addaction name="action_message"/>
    <addaction name="action_shutdown"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_refresh"/>
   <addaction name="action_message"/>
   <addaction name="action_shutdown"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_refresh">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikonka/image/IOS/icons8-синхронизировать-64.png</normaloff>:/ikonka/image/IOS/icons8-синхронизировать-64.png</iconset>
   </property>
   <property name="text">
    <string>обновить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_message">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikonka/image/IOS/icons8-моя-тема-64.png</normaloff>:/ikonka/image/IOS/icons8-моя-тема-64.png</iconset>
   </property>
   <property name="text">
    <string>отправить сообщение</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="action_shutdown">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikonka/image/IOS/icons8-удалить-64.png</normaloff>:/ikonka/image/IOS/icons8-удалить-64.png</iconset>
   </property>
   <property name="text">
    <string>отключить</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
